app/core/utils.coffee

```diff
@@ -215,12 +215,21 @@ if document?.createElement
 module.exports.userAgent = ->
   window.navigator.userAgent
 
-module.exports.getQueryVariable = getQueryVariable = (param, defaultValue) ->
-  query = document.location.search.substring 1
+exports.getDocumentSearchString = ->
+  # moved to a separate function so it can be mocked for testing
+  return document.location.search
+
+module.exports.getQueryVariables = ->
+  query = exports.getDocumentSearchString().substring(1)
```

Comment:
Went ahead and updated it to the new setup instead, putting `module.exports` at the end of the file.